app/components/request-config.js

```diff
@@ -0,0 +1,35 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { later } from '@ember/runloop';
+
+export default Component.extend({
+  copied: false,
+  isExpanded: true,
+
+  toggleStatusClass: computed('isExpanded', function () {
+    return this.get('isExpanded') ? 'expanded' : 'collapsed';
+  }),
+
+  buttonLabel: computed('copied', function () {
+    return this.copied ? 'Copied!' : 'Copy build config';
+  }),
+
+  formattedConfig: computed('config', 'slug', function () {
+    let config = this.get('config');
```

Comment:
Since `config` doesn't seem to be reassigned a value in this function, I would use `const` rather than `let` here. We don't have a set policy on when to use one or the other, so consider this optional, but I generally use `const` unless I need to reassign the variable.